modules/cachedb_local/doc/cachedb_local_admin.xml

```diff
@@ -22,7 +22,15 @@
 	<section>
 		<title>&osips; Modules</title>
 		<para>
-		None.
+		The following modules must be loaded before this module:
```

Comment:
I would suggest adding a small new main chapter describing the clustering related behavior - what are the limitations when acting as a distributed cluster (in terms of consistency and atomic changes), the sync concept (high level). This chapter may cross link with the the new mod params.

Reply:
I tried - I am that good at writing so any input or help is appreciated.

Reply:
What I have in mind here is a short explanation on how the data distribution is done when using the clustering support, like - we simply replicate across the cluster all the local operation. Also the warning as there is no guarantee in terms of data consistency across the cluster, as ops from different node over the same key may be applied in different order on different nodes.
Something like this, so people will not have false expectation from this module

Reply:
Did you check the updated PR?

Reply:
oh, missed that. It looks good, thanks